.github/workflows/codeql-analysis.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [ master ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ master ]
+    branches: [ master, link_tooltip ]
```

Comment:
This change was only for testing if the fix worked, right? You should revert this change now.